config/application.rb

```diff
@@ -20,6 +20,14 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
+    # Enable Cross-Origin Resource Sharing (CORS) from all sources
+    config.middleware.insert_before 0, 'Rack::Cors' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.